app/components/QuestionForm/index.js

```diff
@@ -163,7 +165,9 @@ export const QuestionForm = ({
 
       <TipsBase>
         <BaseSpecialOne>
-          <FormBox onSubmit={handleSubmitWithType(sendQuestion)}>
+          <FormBox onSubmit={(e)=>{
```

Comment:
please not use inline function, create  function for this

Reply:
chenged